release_notes/ocp-4-18-release-notes.adoc

```diff
@@ -2995,6 +2995,63 @@ This section will continue to be updated over time to provide notes on enhanceme
 For any {product-title} release, always review the instructions on xref:../updating/updating_a_cluster/updating-cluster-web-console.adoc#updating-cluster-web-console[updating your cluster] properly.
 ====
 
+// 4.18.8
+[id="ocp-4-18-8_{context}"]
+=== RHSA-2025:3577 - {product-title} {product-version}.8 bug fix update and security update
+
+Issued: 08 April 2025
+
+{product-title} release {product-version}.8 is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHSA-2025:3577[RHSA-2025:3577] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2025:3579[RHBA-2025:3579] advisory.
+
+Space precluded documenting all of the container images for this release in the advisory.
+
+You can view the container images in this release by running the following command:
+
+[source,terminal]
+----
+$ oc adm release info 4.18.8 --pullspecs
+----
+
+[id="ocp-4-18-8-known-issues_{context}"]
+==== Known issues
+
+* IPsec is not supported on {op-system-base-full} compute nodes because of a `libreswan` incompatiblility issue between a host and an `ovn-ipsec` container that exist in each compute node. See (link:https://issues.redhat.com/browse/OCPBUGS-52949[*OCPBUGS-52949*]).
+
+[id="ocp-4-18-8-bug-fixes_{context}"]
+==== Bug fixes
+
+* Previously, sometimes network interface controllers (NICs) attached to virtual machines (VMs) that ran on {azure-first} failed because the NICs were in a `ProvisioningFailed` state. With this release, the Machine API controller now checks the provisioning status of a NIC and refreshes the VMs on a regular basis. If a NIC is in `ProvisioningFailed` state, the VM now fails so that you have a better indication of the issue for troubleshooting purposes. (link:https://issues.redhat.com/browse/OCPBUGS-54355[*OCPBUGS-54355*]) 
+
+* Previously, containers that use the SELinux domain of `container_logreader_t` for the purposes of viewing container logs on a host at `/var/log` could not access logs in the `/var/log/containers` subdirectory. This issue happened because of a missing symbolic link. With this release, a symbolic link is created for `/var/log/containers` so that containers can access the logs in `/var/log/containers`. (link:https://issues.redhat.com/browse/OCPBUGS-54342[*OCPBUGS-54342*]) 
+
+// SME review
+* Previously, selecting the *All projects* option on the *VolumeSnapshot* page of the web console in the *Administrator* perspective resulted in a `404: Page Not Found` error. With this release, a fix ensures that when you select the *All projects* option on the *VolumeSnapshot* page, the page shows results as expected without a `404: Page Not Found` error showing on the page. (link:https://issues.redhat.com/browse/OCPBUGS-54269[*OCPBUGS-54269*])
```

Comment:
```suggestion
* Previously, selecting the *All projects* option on the *VolumeSnapshot* page of the web console in the *Administrator* perspective resulted in a `404: Page Not Found` error. With this release, a fix ensures that when you select the *All projects* option on the *VolumeSnapshot* page, the page shows results as expected without an error. (link:https://issues.redhat.com/browse/OCPBUGS-54269[*OCPBUGS-54269*])
```

IMO the last part can be removed (repeated) because the error is mentioned in the earlier para

Reply:
:heavy_check_mark: